models/filtermodel.cpp

```diff
@@ -24,6 +25,7 @@ FilterModel::FilterModel(QObject *parent)
     : QSortFilterProxyModel(parent)
 {
     this->setFilterRole(DocumentListModel::Roles::FileNameRole);
+    setSortParameter(SortParameter::Date);
```

Comment:
Looks like this gets the oldest first. Think it should be newest.

Reply:
What do you mean?

Reply:
Ok. I'll do it

Reply:
Done